alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/binder/clustering/ClusteringMetrics.java

```diff
@@ -0,0 +1,72 @@
+package eu.xenit.alfred.telemetry.binder.clustering;
+
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+import org.alfresco.enterprise.repo.cluster.core.ClusterService;
+import org.alfresco.enterprise.repo.cluster.core.ClusterServiceImpl;
+
+public class ClusteringMetrics {
+    private static final String CLUSTER_TYPE = "ClusterType";
+    private static final String CLUSTER_NAME = "ClusterName";
+    public static final String GAUGE_NAME = "repository.nodes.count";
+    public static final String GAUGE_DESCRIPTION = "The amount of repository nodes";
+
+    private ClusterServiceImpl clusterService;
+    private MeterRegistry registry;
+
+    public ClusteringMetrics(ClusterServiceImpl clusterService, MeterRegistry registry) {
```

Comment:
Do you need the "Impl" for ClusterServiceImpl in the constructor?

Reply:
Good catch, adjusted